client/web/src/search/home/SearchPageInput.tsx

```diff
@@ -155,7 +160,13 @@ export const SearchPageInput: React.FunctionComponent<React.PropsWithChildren<Pr
                         queryState={userQueryState}
                         onChange={setUserQueryState}
                         onSubmit={onSubmit}
-                        autoFocus={props.showOnboardingTour ? shouldFocusQueryInput : props.autoFocus !== false}
+                        autoFocus={
+                            props.showOnboardingTour
+                                ? shouldFocusQueryInput
+                                : isTouchDevice
+                                ? false
+                                : props.autoFocus !== false
```

Comment:
```suggestion
                            props.showOnboardingTour
                                ? shouldFocusQueryInput
                                : !isTouchDevice && props.autoFocus !== false
```

nit: This is the same, and more readable imo

Reply:
I agree, but I got Prettier warnings previously and then ran prettier to fix it which resulted in the 5 line version. 🤷

Reply:
This also changes it so it's not using a nested ternary and just using boolean evaluation for the non-onboarding-tour path. Prettier shouldn't complain at this one AFAICT

Reply:
Indeed it does! 🤦 
Thanks.